openmdao/code_review/test_lint_peps.py

```diff
@@ -122,4 +122,4 @@ def test_pep257(self):
 
 
 if __name__ == "__main__":
-    unittest.main()
\ No newline at end of file
+    unittest.main()
```

Comment:
Is this a whitespace change?  Doesn't really matter just curious

Reply:
yeah.. missing newline at end of file..  VS Code autocorrected it...